java/core/src/test/java/org/bondlib/RandomRoundTripTest.java

```diff
@@ -0,0 +1,264 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package org.bondlib;
+
+import org.bondlib.test.*;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.EnumMap;
+
+import static org.junit.Assert.*;
+
+public class RandomRoundTripTest {
+
+    private static final int testIterationCount = 1;
```

Comment:
Just noticed this is 1. What's the point of having a tunable and a loop here if we're setting it to 1 now? When would we increase it and do more rounds?

Reply:
When I make it maxint and head into the weekend.

Reply:
What number would you like?